packages/block-library/src/navigation-link/editor.scss

```diff
@@ -66,6 +66,30 @@
 		white-space: nowrap !important; // TODO: solve differently to prevent use of !important
 	}
 
+	//arrow indicator to show presence of children
+	&.has-child {
+		position: relative;
+		padding-right: 15px;
+
+		&::after {
+			content: "";
+			position: absolute;
+			right: 0;
+			top: 50%;
+			border-left: 4px solid transparent;
```

Comment:
Can we consider using an icon or char instead of a border technique? Keep in mind that, at least for now, we have only two ways to apply colors: `color` and `background-color`. Applying border-color will be a pain.
We can use dashicons or even SVG.